cmd/dashboard-token-proxy/main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+type GithubResponse struct {
+	AccessToken string `json:"access_token"`
+	Scope       string `json:"scope"`
+	TokenType   string `json:"token_type"`
+}
+
+type DashboardResponse struct {
+	AccessToken string `json:"access_token"`
+	Success     bool   `json:"success"`
+}
+
+var (
```

Comment:
Consider using struct and refactor HandleTokenRequest to accept the configuration struct instead of using global variables.

type Config struct {
    ClientID         string
    ClientSecret     string
    GhURL            string
    AuthorizationURL string
}

Reply:
passed via separated parameters, as it's more code movement than refactoring let's leave it in that state.